modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -369,12 +419,28 @@ protected function render() {
 				'tabindex' => 1 === $tab_count ? '0' : '-1',
 				'aria-controls' => 'elementor-tab-content-' . $id_int . $tab_count,
 				'aria-expanded' => 'false',
+				'id' => $tab_id,
 			] );
 
+			$this->add_render_attribute( 'icon-selected', 'class', 'elementor-icon-selected' );
+			$this->add_render_attribute( 'icon-active', 'class', 'elementor-icon-active' );
+
 			$title_render_attributes = $this->get_render_attribute_string( $tab_title_setting_key );
 			$mobile_title_attributes = $this->get_render_attribute_string( $tab_title_mobile_setting_key );
+			$tab_icon_attributes = $this->get_render_attribute_string( 'icon-selected' );
+			$tab_icon_active_attributes = $this->get_render_attribute_string( 'icon-active' );
+
+			$icon_html = Icons_Manager::try_get_icon_html( $item['selected_icon'], [ 'aria-hidden' => 'true' ] );
+			$icon_active_html = $icon_html;
+			if ( $this->is_active_icon_exist( $item ) ) {
+				$icon_active_html = Icons_Manager::try_get_icon_html( $item['active_icon'], [ 'aria-hidden' => 'true' ] );
+			}
 
-			$tabs_title_html .= "<div $title_render_attributes>$tab_title</div>";
+			$tabs_title_html .= "<div $title_render_attributes>";
+			$tabs_title_html .= "	<span $tab_icon_attributes> $icon_html</span>";
```

Comment:
Prefer explicit character, for example: `"\t"`

Reply:
Done